Source/Core/Core/HW/EXI/BBA/TAP_Win32.cpp

```diff
@@ -210,8 +210,7 @@ bool CEXIETHERNET::TAPNetworkInterface::Activate()
     INFO_LOG_FMT(SP1, "TAP-Win32 Driver Version {}.{} {}", info[0], info[1],
                  info[2] ? "(DEBUG)" : "");
   }
-  if (!(info[0] > TAP_WIN32_MIN_MAJOR ||
-        (info[0] == TAP_WIN32_MIN_MAJOR && info[1] >= TAP_WIN32_MIN_MINOR)))
+  if (info[0] < TAP_WIN32_MIN_MAJOR)
```

Comment:
This isn't a good change. The condition is there in case the value of `TAP_WIN32_MIN_MINOR` changes from zero. Please eliminate the warning in some other way if that's what you're trying to do. Maybe you can just change `TAP_WIN32_MIN_MINOR` and `TAP_WIN32_MIN_MAJOR` to `constexpr`s instead of macros?